samples/play-kml-tour/src/main/java/com/esri/arcgismaps/sample/playkmltour/screens/PlayKMLTourScreen.kt

```diff
@@ -0,0 +1,136 @@
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.Icon
+import androidx.compose.material3.LinearProgressIndicator
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.mapping.kml.KmlTourStatus
+import com.arcgismaps.toolkit.geoviewcompose.SceneView
+import com.esri.arcgismaps.sample.playkmltour.R
+import com.esri.arcgismaps.sample.playkmltour.components.PlayKMLTourViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialog
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+
+/**
+ * Main screen layout for the sample
+ */
+@Composable
+fun PlayKMLTourScreen(sampleName: String) {
+    val viewModel: PlayKMLTourViewModel = viewModel()
+
+    val state = remember { viewModel.statusFlow }
+    val stateFlow by state.collectAsStateWithLifecycle(KmlTourStatus.NotInitialized)
+
+    val progress = remember { viewModel.progressFlow }
+    val progressFlow by progress.collectAsStateWithLifecycle(0.0f)
+
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        content = {
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(it),
+            ) {
+                SceneView(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .weight(1f),
+                    isAttributionBarVisible = false,
+                    arcGISScene = viewModel.arcGISScene,
+                    sceneViewProxy = viewModel.sceneViewProxy
+                )
+
+                val padding = 8.dp
+
+                LinearProgressIndicator(progress = { progressFlow },
+                    modifier = Modifier.fillMaxWidth().padding(all = padding))
+                Text("${stringResource(R.string.tour_status)} ${tourStateToString(stateFlow)}",
+                    modifier = Modifier.padding(all = padding).align(Alignment.CenterHorizontally))
+                Row(modifier = Modifier.fillMaxWidth()) {
+                    Button(enabled = (stateFlow == KmlTourStatus.Paused || stateFlow == KmlTourStatus.Playing),
+                        modifier = Modifier.fillMaxWidth(0.5f).padding(all = padding), onClick = {
+                        viewModel.reset()
+                    }) {
```

Comment:
We normally make the chain of modifer.padding.fill.etc.etc vertically so more like this:

```suggestion
                LinearProgressIndicator(
                progress = { progressFlow }, modifier = Modifier
                    .fillMaxWidth()
                    .padding(all = padding)
            )
            Text(
                "${stringResource(R.string.tour_status)} ${tourStateToString(stateFlow)}",
                modifier = Modifier
                    .padding(all = padding)
                    .align(Alignment.CenterHorizontally)
            )
            Row(
                modifier = Modifier.fillMaxWidth()
            ) {
                Button(enabled = (stateFlow == KmlTourStatus.Paused || stateFlow == KmlTourStatus.Playing),
                    modifier = Modifier
                        .fillMaxWidth(0.5f)
                        .padding(all = padding),
                    onClick = {
                        viewModel.reset()
                    }) {
```